CONTRIBUTING.md

```diff
@@ -108,29 +108,63 @@ For hassle-free code contribution, follow the steps below. NOTE: I mostly work i
 
 ### Visual Studio Code Development Container
 
-Developing using Visual Studio Code development container lets you use a Docker container as a development environment. The Tern repository contains a definition for a pre-configured Docker container to simplify the setup process. To use it you need:
+Developing using Visual Studio Code development container lets you use a Docker container as a development environment. The Tern repository contains a definition for a pre-configured Docker container to simplify the setup process. To use it you need to download the following software on your host machine:
 
 - [Visual Studio Code](https://code.visualstudio.com/)
 - [Docker](https://www.docker.com/get-started)
 - [The Visual Studio Code Remote - Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
 
-Once you have all that installed you can:
+Once you have all that installed you can setup your dev enviornment:
+
+1. Fork the main Tern repository using your GitHub account. Once forked, clone *your* fork (not the main Tern repo) on your local desktop.
+2. Using the terminal, `cd` into the forked Tern directory on your local desktop and setup your git credentials:
+```
+    git config --global user.name "Your Name"
+    git config --global user.email "your.email@address.com"
+```
+3. Open Visual Studio Code. Click the `><` icon in the lower lefthand corner. A search bar will come up at the top of the window. Type "Remote-Container: Open Folder in Container" and click on it. In the window that pops up, navigate to the cloned `tern` folder on you desktop and click "Open".
```

Comment:
This the remote settings icon, but it's not where you find that command. They need to click on the gear/cog icon. 
```suggestion
3. Open Visual Studio Code. Click the `gear` icon in the lower lefthand corner, or use the `F1` key on your keyboard. This opens the "Command Palette". A search bar will come up at the top of the window. Type "Remote-Container: Open Folder in Container" and click on it. In the window that pops up, navigate to the cloned `tern` folder on you desktop and click "Open".
```